scatter.js

```diff
@@ -565,12 +565,14 @@ Scatter.prototype.update = function (...args) {
 				}
 
 				// update position buffers
-				positionBuffer({
-					data: f32.float(positions),
+				var float_data = f32.float32(positions)
+				state.positionBuffer({
```

Comment:
`state` in not defined!
Fixed in https://github.com/gl-vis/regl-scatter2d/commit/42113ead9fd2a2cf3f4c8043c71ee824dcba47c8